aspnetcore/signalr/supported-platforms.md

```diff
@@ -28,6 +28,8 @@ The [JavaScript client](xref:signalr/javascript-client) runs on NodeJS 8 and lat
 
 &dagger;*Current* refers to the latest version of the browser.
 
+The JavaScript client doesn't support Internet Explorer.
```

Comment:
Maybe add something like.
"and other older browsers. Your mileage may vary if you try other browsers."